variables-skyline/index280.html

```diff
@@ -151,5 +151,5 @@
 </html>
 
 <!--
-  I see some code that can be optimized. Move the `position` and `top` properties and values from `foreground-buildings` to `background-buildings`. Then select both `background-buildings` and `foreground-buildings` there, effectively applying those styles to both of the elements. You can use a comma (`,`) to separate selectors like this: `class1, class2`. Then delete the old `foreground-buildings` class and all its properties.
+  I see some code that can be optimized. Move the `position` and `top` properties and values from `foreground-buildings` to `background-buildings`. Then select both `background-buildings` and `foreground-buildings` there, effectively applying those styles to both of the elements. You can use a comma (`,`) to separate selectors like this: `selector1, selector2`. Then delete the old `foreground-buildings` class and all its properties.
```

Comment:
Would be good to break this down into two steps: moving the properties, then changing the selector to target both foreground and background buildings.